java/build.sh

```diff
@@ -2,6 +2,34 @@
 
 SOURCEDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
 
+
+## revert https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/7970
+
+git clone https://github.com/open-telemetry/opentelemetry-java-instrumentation.git
+pushd opentelemetry-java-instrumentation
+git checkout v1.26.0 -b tag-v1.26.0
```

Comment:
To my understanding, this patch will be applied and works for v1.26.0 but what would our plan look like for future release, i.e. `v1.27.0`?

Reply:
We will need to continue with these patches until a final solution is reached upstream.